source/database/mssql/scripts/dbscripts/PSP_PIMS_S47_00/Alter Up/000_PSP_PIMS_PMBC_SCHEMA_Build.sql

```diff
@@ -0,0 +1,38 @@
+-- Script generated by Aqua Data Studio Schema Synchronization for MS SQL Server 2016 on Tue Oct 05 16:47:03 PDT 2021
+-- Execute this script on:
+-- 		pmbc schema will be created
+
+-- We recommend backing up the database prior to executing the script.
+
+SET XACT_ABORT ON
```

Comment:
@dfilteau can you explain why we need this? I don't remember seeing this in previous alter ups.